### DISCUSSION TITLE
Error: Body exceeded 1mb limit

### DISCUSSION THREAD

**Discussion by cinewar** [blitz-js/blitz#2643] [General]
Hello friends,

I have a project and in this project when I use mutations it gives me this error.

Can anyone help me....

**Comment by flybayer** [comment]
You can increase the body size limit: https://blitzjs.com/docs/api-routes#custom-config

```ts
export const config = {
  api: {
    bodyParser: {
      sizeLimit: "2mb",
    },
  },
}
```
